### PR TITLE
docs: add 3.16.0 upgrade note for import process

### PR DESCRIPTION
**Pull request by marcambier**
https://github.com/gravitee-io/issues/issues/7003
https://github.com/gravitee-io/issues/issues/7013

### CODE REVIEW

File: upgrades/3.x/3.16.0/README.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+= Upgrade to 3.16.0
+
+== API definition import process changes
+
+Gravitee 3.16.0 introduces a new **crossId** in API definition, which identify entities across environments.
```
Comment (phiz71) [inline upgrades/3.x/3.16.0/README.adoc:5]:
```suggestion
Gravitee 3.16.0 introduces a new **crossId** in API definition, which identifies entities across environments.
```

File: upgrades/3.x/3.16.0/README.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+This improves the API import and promotion processes reliability.
+
+If you are using the _API import from JSON definition_ feature,
+we **highly recommend** updating your API definitions by re-exporting your API.
+
+Otherwise, if you import an API definition which doesn't contain **crossId**,
+Gravitee will do the best-effort to import your API definition without it.
+
+Learn more about API definition import link:/apim/3.x/apim_publisherguide_import_apis.html[here].
```
Comment (phiz71) [inline upgrades/3.x/3.16.0/README.adoc:14]: Not sure, this link will work. On top of that, we must ensure that links are relative.
```suggestion
Learn more about API definition import link:{{ '/apim/3.x/apim_publisherguide_import_apis.html' | relative_url}}[here].
```

File: upgrades/3.x/3.16.0/README.adoc [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+== Deprecations
+
+The Rest API endpoints listed below are deprecated since Gravitee v3.0.9, and will be removed in a future version.
```
Comment (phiz71) [inline upgrades/3.x/3.16.0/README.adoc:18]: We could remind the endpoints users should call.

File: upgrades/3.x/README.adoc [hunk 4]
```diff
@@ -1,6 +1,9 @@
 = Upgrade Notes
 
 ifdef::env-github[]
+
+include::3.16.0/README.adoc[leveloffset=+1]
```
Comment (phiz71) [inline upgrades/3.x/README.adoc:5]: should also be added line 96 (for the import in documentation website)

**Review by phiz71 — CHANGES_REQUESTED** [review]

File: upgrades/3.x/3.16.0/README.adoc (hunk 3 above)
Comment (marcambier) [reply upgrades/3.x/3.16.0/README.adoc:18]: it's below :arrow_down:

File: upgrades/3.x/README.adoc (hunk 4 above)
Comment (marcambier) [reply upgrades/3.x/README.adoc:5]: :heavy_check_mark:

**Review by phiz71 — APPROVED** [review]

**Comment by phiz71** [comment]
⚠️ Wait the release to merge this PR ⚠️

**Pull request merged**